Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -147,6 +147,36 @@ function Add-User($authUrl, $name, $accessToken) {
     return Invoke-Post $url $body $accessToken
 }
 
+function Add-UserOrGroupToEdwAdmin($userOrGroup, $connString) {
+	$query = "DECLARE @userId INTEGER;
+			  DECLARE @roleId INTEGER;
+			  IF NOT EXISTS (SELECT * FROM [EDWAdmin].[CatalystAdmin].[IdentityBASE] WHERE [IdentityNM] = @identityName)

```

Comment:
Do we need to handle adding the user if the user doesn't already exist in IdentityBASE?

Reply:
thats what this line is for:

IF NOT EXISTS (SELECT * FROM [EDWAdmin].[CatalystAdmin].[IdentityBASE] WHERE [IdentityNM] = @identityName)

however, if the user exists and they do not have permission, then I need to handle this case

Reply:
Tested this functionality.  It will insert the user if its not there, then it will check to see if that user has access and will add it if they do not.